test/phylogenies.js

```diff
@@ -250,6 +244,149 @@ describe('PhylogenyWrapper', function () {
     });
   });
 
+  describe('#asJSONLD', function () {
+    it('should generate the phylogeny in JSON-LD as expected', function () {
+      const expectedResults = [
```

Comment:
Is there a good reason that this object is declared with all hard-coded IDs instead of using pre-defined identifiers? Test cases should whenever possible  be written such that future changes will either not or exceedingly rarely be needed.